### PR TITLE
 fix transaction IDs in legacy UI trade info views

### PR DESCRIPTION
**Pull request by napoly**

### CODE REVIEW

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -116,6 +116,8 @@ shared.OR=OR
 shared.notEnoughFunds=You don''t have enough funds in your Haveno wallet for this transaction—{0} is needed but only {1} is available.\n\nPlease add funds from an external wallet, or fund your Haveno wallet at Funds > Receive Funds.
 shared.waitingForFunds=Waiting for funds...
 shared.depositTransactionId=Deposit transaction ID
```
Comment (woodser) [inline core/src/main/resources/i18n/displayStrings.properties:118]: I was expecting `shared.yourDepositTransactionId`, `shared.peerDepositTransactionId`, `shared.makerDepositTransactionId` and `shared.takerDepositTransactionId` based on our conversation.

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/ContractWindow.java [hunk 2]
```diff
@@ -265,9 +265,6 @@ private void addContent() {
         addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.makerFeeTxId"), offer.getOfferFeePaymentTxId());
         addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.takerFeeTxId"), "TAKER FEE TX ID NOT PART OF CONTRACT");  // TODO (woodser): should taker fee tx id be part of contract?
 
-        if (dispute.getDepositTxSerialized() != null)
```
Comment (woodser) [inline desktop/src/main/java/bisq/desktop/main/overlays/windows/ContractWindow.java:268]: This removes the deposit tx hash from the contract window. Per our conversation, this view was to show the maker and taker deposit tx hashes.

**Review by woodser — CHANGES_REQUESTED** [review]
See comments.

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/ContractWindow.java (hunk 2 above)
Comment (napoly) [reply desktop/src/main/java/bisq/desktop/main/overlays/windows/ContractWindow.java:268]: Ah I misunderstood. Thought there's going to be a follow up PR. Please check again.

File: core/src/main/resources/i18n/displayStrings.properties (hunk 1 above)
Comment (napoly) [reply core/src/main/resources/i18n/displayStrings.properties]: Removed `depositTransactionId` completely and added taker/maker.

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java [hunk 3]
```diff
@@ -273,13 +268,11 @@ private void addContent() {
                         Res.get(contract.getPaymentMethodId()));
         }
 
-        addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.makerFeeTxId"), offer.getOfferFeePaymentTxId());
-
         if (trade.getMakerDepositTx() != null)
-            addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.depositTransactionId"), // TODO (woodser): separate UI labels for deposit tx ids
+            addLabelTxIdTextField(gridPane, ++rowIndex, Res.get("shared.yourDepositTransactionId"),
```
Comment (woodser) [inline desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java:272]: Should be `shared.makerDepositTransactionId` and `shared.takerDepositTransactionId` here.

**Review by woodser — APPROVED** [review]

**Comment by woodser** [comment]
Thanks :)

**Pull request merged**